packages/commonwealth/client/styles/components/sidebar/CommunitySection.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../shared';
+
+.community-menu {
+  background-color: $neutral-25;
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  min-width: $sidebar-width;
+  overflow-y: auto;
+  width: $sidebar-width;
+
+  @include smallInclusive {
+    width: calc(100vw - $quick-switcher-width);
+  }
+
+  .home-button {
+    align-items: center;
+    display: flex;
+    padding: 8px;
+    cursor: pointer;
+
+    &:hover {
+      background-color: $primary-50;
+    }
+
+    &.active {
+      background-color: $primary-100;
+    }
+
+    svg {
+      margin-right: 4px;
+    }
+  }
+
+  .buttons-container {
+    margin-top: 24px;
+
+    .subscription-button {
+      display: flex;
+      justify-content: center;
+      margin-bottom: 16px;
+    }
+  }
+}
+
+.community-menu-skeleton {
```

Comment:
cc @masvelio is it proper style to keep the skeleton styling alongside the main component, or should it also be separated out into a separate file?

Reply:
well ideally we should keep each component in separate file and each component should have its own scss file

Reply:
Done